src/params.rs

```diff
@@ -207,3 +212,58 @@ impl FromStr for GuestMemorySize {
 		Ok(memory_size)
 	}
 }
+
+/// Configure the kernels environment variables.
+#[derive(Debug, Clone, PartialEq)]
+pub enum EnvVars {
+	/// Pass all env vars of the host to the kernel.
+	Host,
+	/// Pass a certain set of env vars to the kernel.
+	Set(HashMap<String, String>),
+}
+impl Default for EnvVars {
+	fn default() -> Self {
+		Self::Set(HashMap::new())
+	}
+}
+impl<S: AsRef<str> + std::fmt::Debug + PartialEq<S> + From<&'static str>> From<&[S]> for EnvVars {
```

Comment:
This could `impl FromIterator` instead.

Reply:
Ah, no, `From` and `FromIterator` should not panic.

If we want to specialize this for CLI and add graceful error messages, this could [`impl FromArgMatches`](https://docs.rs/clap/latest/clap/trait.FromArgMatches.html).

Reply:
Hmm, yeah. Would be an alternative. But is it worth changing the code?

Reply:
Changed implementation from `From` to `TryFrom` as discussed offline